packages/http-client-csharp/generator/TestProjects/CadlRanch/http/client/structure/multi-client/src/Generated/ClientAClient.RestClient.cs

```diff
@@ -0,0 +1,97 @@
+// <auto-generated/>
+
+#nullable disable
```

Comment:
Just a general comment, we typically do not check in the entire generated library. We only check in the stubbed API version of each file. Here is an existing [example](https://github.com/microsoft/typespec/tree/main/packages/http-client-csharp/generator/TestProjects/CadlRanch/http/parameters/spread/src/Generated). Here is the doc on how to generate the stubbed cadlranch libraries : https://github.com/microsoft/typespec/blob/main/packages/http-client-csharp/generator/docs/cadl-ranch.md#generating-cadlranchtests